ui/src/app/views/account-creation/component.html

```diff
@@ -32,10 +32,12 @@ <h5 style="display: inline-block;">@{{gsuiteDomain}}</h5>
     </div>
 
     <div class="form-section">
-      <button (click)="createAccount()" class="btn btn-primary short-button">Next</button>
-      <span class="spinner spinner-inline" *ngIf="creatingAcccount">
-        Creating account...
-      </span>
+      <button type="submit"
+          class="btn btn-primary short-button"
+          [disabled]="creatingAcccount"
+          [clrLoading]="creatingAcccount">
```

Comment:
Nit: Spelling

Reply:
Huh - good catch. Incredibly this was mispelled everywhere so it actually was working - likely the work of some autocomplete/IDE.